lib/user/follow.js

```diff
@@ -30,7 +30,11 @@ function follow (jar, token, userId) {
           throw new Error(body.message)
         }
       } else {
-        throw new Error('Follow failed')
+        if(body.message){
```

Comment:
body is undefined and doesn't conform to standard. please test your code before submitting pull requests